.vitepress/theme/components/CardGrid.vue

```diff
@@ -23,14 +22,14 @@ defineProps<{ links: any[] }>();
   gap: 24px;
   list-style: none;
 
-  @media (min-width: $bp-medium) {
+  @media (min-width: 680px) {
```

Comment:
Not directly required by this PR: I removed the `.vitepress/theme/styles/vars.scss` file which was defining a few breakpoint values because:

- it defined 3 values and we are using more than that;
- the same 3 values defined in Sass variables were also often hardcoded in other `.scss` files.

Using breakpoints defined in Sass variables was the exception and not the rule, so using hardcoded breakpoint values everywhere makes these styles more consistent and removes a layer of indirection.